<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>761</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>713</width>
    <height>641</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PDF Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>500</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="CustomDragTreeView" name="pdfPagesList">
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragOnly</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ContiguousSelection</enum>
           </property>
          </widget>
          <widget class="PageGridWidget" name="pdfRenderingView" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="CustomDragListView" name="newPagesList">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::CopyAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ContiguousSelection</enum>
       </property>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>410</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="2">
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="pageRangeFirstTxt"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pageRangeLastTxt"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Multiple pages (separated by comma)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="singlePageTxt"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="addSinglePageBtn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="addPageRangeBtn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Single page</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Page Range</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QPushButton" name="addMultiplePagesBtn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="QLineEdit" name="multiplePagesTxt"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load_PDF"/>
    <addaction name="action_Export_PDF"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add_all"/>
    <addaction name="action_Delete_all"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Load_PDF"/>
   <addaction name="action_Export_PDF"/>
   <addaction name="separator"/>
   <addaction name="action_Add_all"/>
   <addaction name="action_Delete_all"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Load_PDF">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/res/icons8_opened_folder_96px.png</normaloff>:/icons/res/icons8_opened_folder_96px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load PDF</string>
   </property>
  </action>
  <action name="action_Export_PDF">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/res/icons8_export_pdf_96px.png</normaloff>:/icons/res/icons8_export_pdf_96px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export PDF</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Add_all">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/res/icons8_add_list_96px.png</normaloff>:/icons/res/icons8_add_list_96px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add all</string>
   </property>
  </action>
  <action name="action_Delete_all">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/res/icons8_delete_property_96px.png</normaloff>:/icons/res/icons8_delete_property_96px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete all</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomDragListView</class>
   <extends>QListView</extends>
   <header>customdraglistview.h</header>
  </customwidget>
  <customwidget>
   <class>CustomDragTreeView</class>
   <extends>QTreeView</extends>
   <header>customdragtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>PageGridWidget</class>
   <extends>QWidget</extends>
   <header>pagegridwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
